### DISCUSSION TITLE
Disable Automatic Email Sending

### DISCUSSION THREAD

**Discussion by c-obsesie** [supabase/supabase#22246] [Questions]
Can I turn off the automatic email dispatch feature in Supabase?

 I prefer handling email sending on my own, such as confirmation emails and similar communications.

I tried by modifying SMTP with a faulty one but did not work

**Comment by suamai** [comment]
Any updates on this?

**Comment by stefan-girlich** [comment]
There does not seem to be a straight-forward solution as of now, see this thread: https://www.reddit.com/r/Supabase/comments/1c5cemm/best_way_to_disable_builtin_mail_sending/

Open (unanswered) feature request here:
https://github.com/orgs/supabase/discussions/18528
